xapi/profiles/.htaccess

```diff
@@ -8,10 +8,12 @@ RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
 RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
 RewriteRule ^cmi5/CourseStructure$ https://github.com/AICC/CMI-5_Spec_Current [R=303]
 RewriteRule ^cmi5/CourseStructure/$ https://github.com/AICC/CMI-5_Spec_Current/ [R=303]
-RewriteRule ^cmi5/CourseStructure.xsd$ http://xapi.vocab.pub/profiles/cmi5/CourseStructure.xsd [R=303]
+RewriteRule ^cmi5/CourseStructure.xsd$ https://raw.githubusercontent.com/AICC/CMI-5_Spec_Current/quartz/v1/CourseStructure.xsd [R=303]
+RewriteRule ^cmi5/v1/CourseStructure.xsd$ https://raw.githubusercontent.com/AICC/CMI-5_Spec_Current/quartz/v1/CourseStructure.xsd [R=303]
 
 # Rewrite rule to serve XML content from the vocabulary URI if requested
 RewriteCond %{HTTP_ACCEPT} application/xml
-RewriteRule ^cmi5/CourseStructure$ http://xapi.vocab.pub/profiles/cmi5/CourseStructure.xsd [R=303]
-RewriteRule ^cmi5/CourseStructure/$ http://xapi.vocab.pub/profiles/cmi5/CourseStructure.xsd [R=303]
-
+RewriteRule ^cmi5/CourseStructure$ https://raw.githubusercontent.com/AICC/CMI-5_Spec_Current/quartz/v1/CourseStructure.xsd [R=303]
+RewriteRule ^cmi5/v1/CourseStructure$ https://raw.githubusercontent.com/AICC/CMI-5_Spec_Current/quartz/v1/CourseStructure.xsd [R=303]
+RewriteRule ^cmi5/CourseStructure/$ https://raw.githubusercontent.com/AICC/CMI-5_Spec_Current/quartz/v1/CourseStructure.xsd [R=303]
+RewriteRule ^cmi5/v1/CourseStructure/$ https://raw.githubusercontent.com/AICC/CMI-5_Spec_Current/quartz/v1/CourseStructure.xsd [R=303]
```

Comment:
That's the theory. :-)  It's the regex way of saying previous element has zero or one occurrences.  In this case, just the "/" character.